Source/Core/Core/PowerPC/JitArm64/JitArm64_Branch.cpp

```diff
@@ -97,11 +90,6 @@ void JitArm64::WriteBranchWatch(u32 origin, u32 destination, UGeckoInstruction i
                    branch_watch, Core::FakeBranchWatchCollectionKey{origin, destination}, inst.hex);
```

Comment:
Replace `branch_watch` here with `&m_branch_watch`. You can then get rid of the MOVP2R at the beginning of the function as well as the `reg_a` parameter. (`ABI_CallFunction` will automatically call MOVP2R for you if you do this. This is the same optimization as you've done in Jit64.)